morango/apps.py

```diff
@@ -1,12 +1,31 @@
 from __future__ import unicode_literals
 
+import logging as logger
+
 from django.apps import AppConfig
+from django.db.utils import OperationalError
 from morango.utils.syncing_utils import add_syncing_models
 
+logging = logger.getLogger(__name__)
+
 
 class MorangoConfig(AppConfig):
     name = 'morango'
     verbose_name = 'Morango'
 
     def ready(self):
+        from morango.models import DatabaseIDModel, InstanceIDModel
+
+        # NOTE: Warning: https://docs.djangoproject.com/en/1.10/ref/applications/#django.apps.AppConfig.ready
+        # its recommended not to execute queries in this method, but we are producing the same result after the first call, so its OK
+
+        # call this on app load up to get most recent system config settings
+        try:
+            if not DatabaseIDModel.objects.all():
+                DatabaseIDModel.objects.create()
+            InstanceIDModel.get_or_create_current_instance()
+        except OperationalError:
```

Comment:
Since the `ready` method in apps.py is called when the app is loaded, and during management commands, if there is no initial database, then the above queries will fail. This was a workaround to allow it to `migrate` a database without making any queries.